app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -1,12 +1,13 @@
 package com.example.ithaca_transit_android_v2.states
 
+import com.example.ithaca_transit_android_v2.Models.LocationType
 import com.example.ithaca_transit_android_v2.models.Coordinate
 import com.example.ithaca_transit_android_v2.models.Location
 
 // Parent Class
 sealed class SearchState(
-    val startLocation: Location = Location("", Coordinate(0.0, 0.0), ""),
-    val endLocation: Location = Location("", Coordinate(0.0, 0.0), "")
+    val startLocation: Location = Location(LocationType.BUSSTOP,"", Coordinate(0.0, 0.0), ""),
```

Comment:
remind me why we set default values for these params again?

Reply:
If I don't create defaults, whenever I want to switch to another state, I am forced to pass in all the parameters. I felt that would be necessary for states like launchState, and Initial search states.